pkgs/development/lua-modules/nfd/default.nix

```
@@ -1,5 +1,5 @@
-{ fetchFromGitHub, buildLuarocksPackage, lua, pkg-config, lib
-, substituteAll, zenity }:
+{ stdenv, fetchFromGitHub, buildLuarocksPackage, lua, pkg-config, lib
+, substituteAll, zenity, darwin }:
```

```suggestion
, substituteAll, zenity, AppKit }:
```

then, add `inherit (darwin.apple_sdk.frameworks) AppKit;` in `all-packages.nix`.

---

Force-pushed to adopt.